sdk/core/azure-core-tracing-opentelemetry/tests/test_tracing_implementations.py

```diff
@@ -26,30 +27,30 @@ def test_span_passed_in(self, tracer):
             wrapped_span = OpenTelemetrySpan(parent)
 
             assert wrapped_span.span_instance.name == "parent"
-            assert parent is tracer.get_current_span()
-            assert wrapped_span.span_instance is tracer.get_current_span()
+            assert parent is trace.get_current_span()
```

Comment:
tracer is a fixture, if you need to change soimething you need to change the fixture, not the test
https://github.com/Azure/azure-sdk-for-python/blob/38048c6da64b2784a427fbec2ad6e499755cc54f/sdk/core/azure-core-tracing-opentelemetry/tests/conftest.py#L12